From 6697f5d63ec79a38baf4fb585eab5ee1fbc1c437 Mon Sep 17 00:00:00 2001
From: Min He <min.he@intel.com>
Date: Tue, 28 Mar 2017 01:37:03 +0800
Subject: [PATCH 0925/2367] drm/i915/gvt: handle BXT in
 dp_aux_ch_ctl_mmio_write

Signed-off-by: Min He <min.he@intel.com>
(cherry picked from commit 58b5112009611b5d78282d33e278ccd6850a1017)

Change-Id: I31c7f1d5da366b28fdc7958318a0009302ba5ccd
Reviewed-on: https://android.intel.com/613348
Reviewed-by: He, Min <min.he@intel.com>
Reviewed-by: Jiang, Fei <fei.jiang@intel.com>
Reviewed-by: Dong, Eddie <eddie.dong@intel.com>
Tested-by: Dong, Eddie <eddie.dong@intel.com>
---
 drivers/gpu/drm/i915/gvt/handlers.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/i915/gvt/handlers.c b/drivers/gpu/drm/i915/gvt/handlers.c
index 475aa8d..5a53a45 100755
--- a/drivers/gpu/drm/i915/gvt/handlers.c
+++ b/drivers/gpu/drm/i915/gvt/handlers.c
@@ -864,6 +864,7 @@ static int dp_aux_ch_ctl_mmio_write(struct intel_vgpu *vgpu,
 	data = vgpu_vreg(vgpu, offset);
 
 	if ((IS_SKYLAKE(vgpu->gvt->dev_priv)
+		|| IS_BROXTON(vgpu->gvt->dev_priv)
 		|| IS_KABYLAKE(vgpu->gvt->dev_priv))
 		&& offset != _REG_SKL_DP_AUX_CH_CTL(port_index)) {
 		/* SKL DPB/C/D aux ctl register changed */
-- 
2.7.4

